_episodes/03-transform-data.md

```diff
@@ -537,7 +537,7 @@ replace variable_name = "population" if indicatorcode == "SP.POP.TOTL"
 drop indicatorcode indicatorname
 reshape wide v, i(countrycode year) j(variable_name) string
 rename v* *
-save "data/WDI-select-variables.dta"
+save "data/WDI-select-variables.dta", replace
```

Comment:
@ariedamuco This omits the `replace` option on purpose so that when they run the .do file again in Episode 5, they learn from the error message that is is safe to `replace` when using .do files. Please remove.

Reply:
@korenmiklos it's in line with the screenshot in the command-history.png. I can easily fix it but the screenshots need to get updated (or we can update Episode 5). In my opinion Episode, 5 needs to be linked with 4, so one of the .do that does merge or append can be used instead. Please see issue #43... Let me know.

Reply:
I think this is well explained in E05 https://github.com/datacarpentry/stata-economics/blame/master/_episodes/05-programming.md#L75 I will leave the command without `replace` so that we produce the intended error.